en/app.ftl

```diff
@@ -276,10 +276,15 @@ landing-pricing-offer-end-cta = Upgrade now
 #   $end_date (string) - The localised date the introductory pricing offer ends, e.g. "September 27 2022"
 landing-pricing-offer-end-body = Get { -brand-name-relay-premium } before { $end_date } and enjoy premium email protection at our intro month-to-month price.
 
+# Deprecated
```

Comment:
just checking - deprecated refers to the line below, right?

Reply:
Correct. A comment is followed by the string below it.